src/App.js

```diff
@@ -0,0 +1,23 @@
+import { Outlet, Route, Routes } from 'react-router-dom';
+import DetailsPage from './components/DetailsPage';
+import HomePage from './components/HomePage';
+import './App.css';
+
+const Layout = () => (
+  <>
+    <Outlet />
+  </>
+);
+
+function App() {
+  return (
+    <Routes>
+      <Route path="/" element={<Layout />}>
+        <Route index element={<HomePage />} />
+        <Route path="/details" element={<DetailsPage />} />
```

Comment:
- [ ] Each item on the home page should have a unique route. Right now, all your items have the same route: `/details`, but they should be something like so: `/details/:id`. Kindly fix that.

  For example (you don't need to use the same word), instead of all your routes have the same routes like so:

  ```
  /details
  ```
  They can have this:
  ```
  /details/angola
  ---
  /details/niger
  ```